recipes-core/images/rdk-generic-broadband-image.bbappend

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
I think you should add `BB_DANGLINGAPPENDS_WARNONLY` configuration as well to keep the possibility to use this layer with poky and without rdk-b

Reply:
`BB_DANGLINGAPPENDS_WARNONLY` has been added into 'conf/layer.conf'